annotation-utils/src/main/java/edu/isi/vista/annotationutils/PushAnnotations.kt

```diff
@@ -92,11 +101,26 @@ fun main(argv: Array<String>) {
         null
     }
 
+    // Build params for extracting the annotation statistics
     val extractAnnotationStatsParamsBuilder = Parameters.builder()
     extractAnnotationStatsParamsBuilder.set("exportedAnnotationRoot", "$localWorkingCopyDirectory" + params.getString("exportedAnnotationRoot"))
     extractAnnotationStatsParamsBuilder.set("statisticsDirectory", params.getString("statisticsDirectory"))
     val extractAnnotationStatsParams = extractAnnotationStatsParamsBuilder.build()
 
+    // Assemble params for converting the JSON to FlexNLP documents
+    val inputAnnotationJsonDir = File("$localWorkingCopyDirectory" + params.getString("exportedAnnotationRoot"))
```

Comment:
Here and elsewhere, don't use `getString` when something more specific like `getExistingFile` or `getCreatableDirectory` would apply.